install/outline-install.sh

```diff
@@ -0,0 +1,157 @@
+echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_${NODE_MAJOR}.x nodistro main" >/etc/apt/sources.list.d/nodesource.list
+msg_ok "Set up Node.js Repository"
+
+msg_info "Setting up Redis Repository"
+wget -qO- https://packages.redis.io/gpg | gpg --dearmor -o /etc/apt/keyrings/redis-archive-keyring.gpg
+echo "deb [signed-by=/etc/apt/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" >/etc/apt/sources.list.d/redis.list
+msg_ok "Set up Redis Repository"
+
+msg_info "Installing Node.js/Yarn"
+$STD apt-get update
+$STD apt-get install -y nodejs
+$STD npm install -g yarn
+msg_ok "Installed Node.js/Yarn"
+
+msg_info "Installing Redis"
+$STD apt-get install -y redis
+sed -i 's/^bind .*/bind 0.0.0.0/' /etc/redis/redis.conf
+systemctl enable -q --now redis-server.service
+msg_ok "Installed Redis"
+
+msg_info "Setting up PostgreSQL DB"
+SECRET_KEY="$(openssl rand -hex 32)"
+UTILS_SECRET="$(openssl rand -hex 32)"
+DB_NAME=outlinedb
+DB_USER=outline
+DB_PASS="$(openssl rand -base64 18 | tr -d '/' | cut -c1-13)"
+DATABASE_URL="postgres://${DB_USER}:${DB_PASS}@localhost:5432/${DB_NAME}"
+$STD sudo -u postgres psql -c "CREATE ROLE $DB_USER WITH LOGIN PASSWORD '$DB_PASS';"
+$STD sudo -u postgres psql -c "ALTER ROLE $DB_USER WITH CREATEDB;"
+$STD sudo -u postgres psql -c "ALTER ROLE $DB_USER SET timezone TO 'UTC';"
+
+{
+    echo "Outline-Credentials"
+    echo "Outline Database User: $DB_USER"
+    echo "Outline Database Password: $DB_PASS"
+    echo "Outline Database Name: $DB_NAME"
+    echo "Outline Secret: $SECRET_KEY"
+    echo "Outline Utils Secret: $UTILS_SECRET"
+} >> ~/outline.creds
+msg_ok "Set up PostgreSQL DB"
+
+
+if [[ "${adminer_prompt,,}" =~ ^(y|yes)$ ]]; then
+  msg_info "Installing Adminer"
+  $STD apt install -y adminer
+  $STD a2enconf adminer
+  systemctl reload apache2
+  IP=$(hostname -I | awk '{print $1}')
+  {
+      echo ""
+      echo "Adminer Interface: $IP/adminer/"
+      echo "Adminer System: PostgreSQL"
+      echo "Adminer Server: localhost:5432"
+      echo "Adminer Username: $DB_USER"
+      echo "Adminer Password: $DB_PASS"
+      echo "Adminer Database: $DB_NAME"
+  } >> ~/outline.creds
+  msg_ok "Installed Adminer"
+fi
+
+msg_info "Installing $APPLICATION (Patience)"
+cd /opt
+RELEASE=$(curl -s https://api.github.com/repos/outline/outline/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+wget -q "https://github.com/outline/outline/archive/refs/tags/${RELEASE}.zip"
+unzip -q ${RELEASE}.zip
+mv outline-${RELEASE:1} /opt/outline
+cd /opt/outline
+
+## First build in development
+$STD yarn install --no-optional --frozen-lockfile
+$STD yarn cache clean
+$STD yarn build
+
+## Continue in production
+rm -rf ./node_modules
+$STD yarn install --production=true--frozen-lockfile
+$STD yarn cache clean
+
+FILE_STORAGE_LOCAL_ROOT_DIR=/var/lib/outline/data
+mkdir -p $FILE_STORAGE_LOCAL_ROOT_DIR
```

Comment:
Why 2 commands?

Reply:
For later use in writing the `.env` file